deep-learning/_posts/2017-04-10-deepmask.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: review
+title:  "Learning to Segment Object Candidates"
+tags:   deep-learning CNN segmentation
+author: Frédéric Branchaud-Charron
+pdf:   "https://arxiv.org/pdf/1506.06204.pdf"
+cite:
+  authors: "Pinheiro, P. O., Collobert, R., & Dollar, P."
+  title:   "Learning to segment object candidates"
+  venue:   "Advances in Neural Information Processing Systems (pp. 1990-1998) (2015)"
+---
+
+This paper presents the DeepMask architecture, which can be used as an object detection method when combined with R-CNN.
+
+DeepMask works on a patch-based approach. For each patch, the model tries to predict two things. First of all, the model will output a pixel-wise segmentation map to find salient objects. Then, in another branch, the model will predict if the patch is at the center of the object. This allows overlapping objects to be segmented correctly.
```

Comment:
will output -> outputs
will predict -> predicts

this allows overlapping objects of the same class to be segmented independently.